src/TextInput/PasswordInput.svelte

```diff
@@ -125,6 +124,7 @@
       class:bx--text-input--light="{light}"
       class:bx--text-input--invalid="{invalid}"
       class="{size && `bx--text-input--${size}`}"
+      {...$$restProps}
```

Comment:
`{...$$restProps}` has to go before `"class="{size && `bx--text-input--${size}`}"` to avoid overriding it.

Reply:
done